WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,227 @@
+
+To help prepare the WEP draft, we encourage community members to discuss the idea publicly, for example in the Slack group, and gather initial community feedback before writing a full WEP. Public discussions on pre-proposals also help to make sure the idea is applicable to the entire community and not just to the author.
+
+A WEP can be co-authored by more than one community member, but we recommend keeping the number of co-authors small and choosing a representative who will submit the PR to the WEPs repository on behalf of the co-authors.
+
+Draft submission and pre-review
+...............................
+
+A draft WEP must be submitted in a GitHub pull request to the `writethedocs/weps <https://github.com/writethedocs/weps>`_ repository, in the ``drafts/`` directory. If you need help with working with GitHub or creating a PR, please feel free to send an e-mail to support@writethedocs.org, and the Write the Docs team will help you get started.
+
+The WEP draft must follow these guidelines:
+
+- WEP is actionable and implementable. The proposed change must be specific and the resulting action clear.
+- WEP fits the scope of the WEP process, as described in **Scope of WEPs**. WEPs that are out of scope of the process will be rejected by the reviewer.
+- Content format must follow the WEP style as described in the **WEP format and style** section. This includes language, grammar, structure, and markup. The PR might be blocked until proper formatting rules are applied.
+- The WEP content is complete. If you want to submit a Work in Progress (WIP) draft before it is ready for pre-review, you can mark the PR as ``WIP`` and the review team will wait until the draft is ready.
+- **Allow edits from maintainers** option is selected. This helps the reviewer merge your WEP draft if it is accepted.
+- The PR is tagged with a **draft** flag, to indicate that it is in a draft state and is ready for pre-review. 
+
+
+After the pull request for the WEP draft is submitted, a WEP team member reviews the pull request to make sure it adheres to the guidelines. If the WEP is clearly not ready, the reviewer might reject the pull request and ask the author to submit a new WEP after the problems have been fixed.
+
+The reviewer does not vote on the proposed change itself, and only reviews the WEP for scope compliance, format, and completeness. The pre-review period is usually one week, but might change based on the scope of the pre-review feedback and the availability of the author to implement changes.
+
+After the WEP is ready for the community review, the reviewer assigns the **ready-for-discussion** label to the PR. The CoreOps team is then responsible for distributing announcements of each WEP to the relevant community segment. For example, a WEP about meetups will be shared with the Meetup team for distribution between the meetup organizers. For some WEPs, community-wide announcements will be made on Slack or to the mailing list. 
```

Comment:
Would this be the "WEP team" now or in the future?

Reply:
Not necessarily, because the WEP team might not have the same access permissions to share the post in admin-only channels on Slack etc..

Reply:
I could see it moving to the WEP team at some point, but mostly just did it this way to simplify the process for now. I think the communication bits are going to be tricky, and probably need the most iteration as we implement things.